README.md

```diff
@@ -1,3 +1,11 @@
 # Continous Deployment
 
 This repository contains an opinionated reference architecture to setup, manage and operate a continous deployment pipeline.
+
+### Prerequisites
+Kustomize 3.8.1+
+SOPS 3.4.0+
+KSOPS 2.1.2+
+
+Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team for access.
```

Comment:
remove this sentence.
Or can we add the plain passwords since it's only for a `dev` namespace?